src/index.html

```diff
@@ -29,18 +29,17 @@
         
         <p><strong>12/01/2016</strong></p>
         <p>- Le site web de la-rache.com sort du néant pour votre 
-        plus grand plaisir !</p>
+        plus grand plaisir&nbsp;!</p>
 
         <p><strong>08/06/2010</strong></p>
         <p>- Première mise à jour depuis une éternité. Quelques
-        nouvelles captures d'écran et un lien. Merci à Thierry
-        !<br /></p>
+        nouvelles captures d'écran et un lien. Merci à Thierry&nbsp;!<br /></p>
 
         <p><strong>22/10/2007</strong></p>
         <p>- Suite à une forte demande populaire d'une personne, la
         RACHE passède désormais un bouton 'hype web2.0'</p>
         <p><img src="img/logo.png" alt="la-rache" /></p>
-        <p>ça va increaser la popularité du site !<br /></p>
+        <p>ça va booster la popularité du site !<br /></p>
```

Comment:
On retrouve aussi ce verbe [ici](https://github.com/la-rache/la-rache.com/blob/master/src/index9.html#L4). *increaser* fait clairement plus pro pour l’*International Institute of La RACHE* :smile:.